src/ResourceManager/DataFactories/Commands.DataFactoryV2/Changelog.md

```diff
@@ -19,5 +19,8 @@
 -->
 
 ## Current Release
+* Added 2 new cmdlets: Update-AzureRmDataFactoryV2 and Stop-AzureRmDataFactoryV2PipelineRun
+
+## 2017.09.25 - Version 0.3.0 (Previous Release)
```

Comment:
No need to add this new header, it will be done automatically in the next release. Just keep everything under the current release header.

Reply:
Done